config/sgp4_scanner.json.empty

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2014-2016 Kartik Kumar (me@kartikkumar.com)
+// Copyright (c) 2014-2016 Abhishek Agrawal (abhishek.agrawal@protonmail.com)
+// Distributed under the MIT License.
+// See accompanying file LICENSE.md or copy at http://opensource.org/licenses/MIT
+
+// Configuration file for D2D "sgp4_scanner" application mode.
+{
+    "mode"                      : "sgp4_scanner",
+
+    // Set path to TLE catalog file.
+    "catalog"                   : "",
```

Comment:
Can be removed, and all instances of the catalog path in the code can be removed.